### PR TITLE
Make `toHexChainID` case-insensitive.

### PR DESCRIPTION
**Pull request by 0xDaedalus**
### To Test
- [x] connect to galxe.com
- [x] Attempt to switch to the optimism network
- [x] You should be able switch to the optimism network

### CODE REVIEW

File: background/tests/networks.unit.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { toHexChainID } from "../networks"
+
+describe("Networks", () => {
+  describe("toHexchainID", () => {
+    it("should be case-insensitive", async () => {
+      expect(toHexChainID("0xA")).toEqual(toHexChainID("0xa"))
```
Comment (greg-nagy) [inline background/tests/networks.unit.test.ts:6]: Hell yeah ❤️

**Review by greg-nagy — APPROVED** [review]
works like a charm ✨

**Comment by Shadowfiend** [comment]
What case-sensitive piece was failing here, do we know?

**Comment by greg-nagy** [comment]
Galxe sent `0xA` as chainID when using the in dapp network switcher.

**Pull request merged**